testing/junit/src/main/java/com/linecorp/armeria/testing/junit/server/mockwebserver/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * A mock server for use when testing clients.
+ */
+@NonNullByDefault
+package com.linecorp.armeria.testing.junit.server.mockwebserver;
```

Comment:
How about just `mock`? Because Armeria is for web and the parent package name indicates it's for server-side.